### PR TITLE
Update HOW-TO.md

### PR DESCRIPTION
**Pull request by burkeholland**
Add XML for SegmentedBar

### CODE REVIEW

File: ApiReference/ui/segmented-bar/HOW-TO.md [hunk 1]
```diff
@@ -8,6 +8,15 @@ Using a SegmentedBar requires the "ui/segmented-bar" module.
 ``` JavaScript
 var segmentedBarModule = require("ui/segmented-bar");
 ```
+```` xml
+<SegmentedBar>
+  <SegmentedBar.items>
+    <SegmentedBarItem title="Item 1">
```
Comment (ErjanGavalji) [inline ApiReference/ui/segmented-bar/HOW-TO.md]: SegmentedBarItem tags need be closed

**Comment by ErjanGavalji** [comment]
The HowTos get generated from snippets in the [unit-test files of the cross-platform modules](https://github.com/NativeScript/NativeScript/blob/master/apps/tests/ui/segmented-bar/segmented-bar-tests.ts). I'm applying it there.

Still, :+1: for merging the change here (after closing the `SegmentedBarItem` tags) to have it appear quicker to the Live documentation.

Please, cherry-pick to `master` branch once merged.

**Comment by burkeholland** [comment]
Done

**Comment by ErjanGavalji** [comment]
:+1:

**Pull request merged**